src/dex-ui/pages/CreateProposal/ProposalForms/TextProposalForm.tsx

```diff
@@ -41,8 +42,27 @@ function TextProposalForm() {
           />
           <FormErrorMessage>{errors.title && errors.title.message}</FormErrorMessage>
         </FormControl>
-        <FormControl>
-          <TextEditor id="description" placeholder="Description" />
+        <FormControl isInvalid={Boolean(errors.description)}>
+          <Controller
+            name="description"
+            control={control}
+            rules={{
+              required: { value: true, message: "Description is required." },
+              minLength: { value: 107, message: "Please enter atleast 100 characters in the description." },
```

Comment:
We used 107 characters here the reason being on the empty state of the editor it still holds the empty tags like <p> and other which counts to 7 so for now keeping the characters validation 100 which makes 100 + 7 to 107.